packages/manager/apps/pci-block-storage/src/api/hooks/useCatalog.ts

```diff
@@ -81,7 +80,8 @@ export const useVolumeRegions = (projectId: string) => {
 export type TVolumeModel = TModelPrice &
   TModelAvailabilityZones &
   TModelName &
-  TModelAttach;
+  TModelAttach &
+  TModelPreselection;
```

Comment:
Here it is not possible to have these values so I don't think the mapVolumeCatalog should be common and have this typing

These are 2 different use cases